commons/dev/configmaps/kafka-configmap.yaml

```
@@ -13,6 +13,7 @@ data:
   CATALOG_TOPIC: "event-store.dev_catalog.events"
   DELEGATION_OUTBOUND_TOPIC: "outbound.dev_delegation.events"
   DELEGATION_TOPIC: "event-store.dev_delegation.events"
+  EMAIL_SENDER_TOPIC: "dev_email-sender.emails"
```

```suggestion
  EMAIL_DISPATCH_TOPIC: "dev_email-notification-dispatcher.emails"
```
We usually put the "source" before the dot. References to this key will need to be updated accordingly